docs/config.ini

```diff
@@ -2,10 +2,14 @@
 nick=OneBotTest
 username=TwoBot
 host=my.irc.network
+
 port=6697
 ssl=true
+ssl_verify = CERT_NONE
```

Comment:
I'd rather have this be a comment: I don't like having insecure behaviour in examples turned on by default.